saige_assoc.py

```diff
@@ -341,12 +347,34 @@ def manage_concurrency_for_job(job: hb.batch.job.Job):
             job.depends_on(jobs[-get_config()['saige']['max_parallel_jobs']])
         jobs.append(job)
 
+    # define writeout file by type of pipeline and date and time
+    date_and_time = datetime.today().strftime('%Y-%m-%d_%H:%M:%S')
+    writeout_file = (
+        f'{writeout_file_prefix}/saige_qtl_common_variant_pipeline_{date_and_time}.csv'
```

Comment:
```suggestion
        f'{writeout_file_prefix}/saige_qtl_common_variant_pipeline_{date_and_time}.json'
```

sorry, this one is on me. The JSON library won't have any issues writing to a `.csv` file, but it will be misleading.